packages/mdc-floating-label/_mixins.scss

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+@import "./variables";
+
+// Public
+
+@mixin mdc-floating-label-color($color) {
+  .mdc-floating-label {
```

Comment:
This one is tricky...

We usually try to avoid putting _block_ CSS selectors in mixins because it violates BEM.

However, floating label is kind of a weird case, because although the class name looks like a block, it really acts more like a sub-element...

Let's chat about this with the rest of the team on Slack and see what they think.

Ditto for the mixins below.

Reply:
looked at line ripple, which just removes the .mdc-line-ripple from its mixin. This should also work for floating-label